lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -637,6 +637,11 @@ private int updateLastVersionsInData(String indexName, GUID parentGUID,
         if (indexName == null) {
```

Comment:
Actually should note that this method now forces a refresh as well. This is used (AFAICT) in indexObjects and undeleteAllVersions.

Reply:
Added comment with reason for the refresh.